src/Stratis.Bitcoin.Features.Consensus/CoinViews/CachedCoinView.cs

```diff
@@ -161,7 +162,7 @@ private CachedCoinView(IDateTimeProvider dateTimeProvider, ILoggerFactory logger
         }
 
         /// <inheritdoc />
-        public override async Task<FetchCoinsResponse> FetchCoinsAsync(uint256[] txIds)
+        public override async Task<FetchCoinsResponse> FetchCoinsAsync(uint256[] txIds, CancellationToken ct = default(CancellationToken))
```

Comment:
We rather use long names like cancellationToken instead of ct, consider replacement, although it is not required

Reply:
I generally go long names, so yes I will change it no big deal

Reply:
Yes, please do everywhere, but as mentioned, I would still prefer this to be on separate PR and not on the PR that should fix a vulnerability